### PR TITLE
FUSETOOLS-3170 - add Fuse 7.2 GA to the new project wizard, add camel…

### PR DESCRIPTION
**Pull request by lhein**
… catalog impl

Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [x] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by apupier** [comment]
7.2 artefacts are availabl on repositories, so there should be no more error in builds

**Comment by lhein** [comment]
Yeah, somehow the versions I gathered from Jira do not match what has been deployed to the repo. Need to investigate.

File: configuration/ReadMe.md [hunk 1]
```diff
@@ -18,42 +23,102 @@ All versions not mentioned here will not have BOM support and just set the Camel
...
+Example:
+	
+	2.21.0.fuse-720050-redhat-00001=7.2.0.fuse-720020-redhat-00001
+	
+So whenever the user creates a project for Camel version 2.21.0.fuse-720050-redhat-00001 the tooling will automatically choose the bom version 7.2.0.fuse-720020-redhat-00001 and set that in the projects pom.xml file.
+All versions not mentioned here will not have BOM support and just set the Camel version directly on the dependencies.
+
+
+## camel2bom.fuse71.properties
+see camel2bom.fuse7.properties above
```
[the remark this answers is not in the document]
Comment (lhein) [reply configuration/ReadMe.md]: can you maybe prepare a PR for that?

[the remark this answers is not in the document]
Comment (apupier) [reply configuration/ReadMe.md]: created https://github.com/jbosstools/jbosstools-fuse/pull/1502

**Comment by apupier** [comment]
Can you restore your branch please? 
it is needed until the org.fusesource.ide.foundation.core.util.OnlineVersionMapper.BASE_REPO_CONFIG_URI has been switched back to the master

**Pull request merged**